### ISSUE TITLE
Http status 406 when requesting actuator endpoint with Feign BOM RC1

### ISSUE THREAD

**Issue by jjroldan1** [spring-cloud/spring-cloud-openfeign#21]
Hi,

I have found this strange behavior while working with Spring Boot 2.0.0.RELEASE and Spring Cloud Finchley.RC1 and Spring Cloud OpenFeign 2.0.0.RC1 with the following maven setup:

```
    <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
         <version>2.0.0.RELEASE</version>
         <relativePath />
    </parent>

    <dependencyManagement>
      <dependencies>
        <dependency>
            <groupId>org.springframework.cloud</groupId>
            <artifactId>spring-cloud-dependencies</artifactId>
            <version>Finchley.RC1</version>
            <type>pom</type>
            <scope>import</scope>
        </dependency>
        <dependency>
            <groupId>org.springframework.cloud</groupId>
            <artifactId>spring-cloud-openfeign</artifactId>
            <version>2.0.0.RC1</version>
            <type>pom</type>
            <scope>import</scope>
        </dependency>
    </dependencies>
</dependencyManagement>    
```

A curl request like 

    curl http://localhost:8081/actuator/health

which was previously working fine, it started returning a HTTP status 406.

Only if I add an Accept header

    curl -H "Accept: */*;q=1" http://localhost:8081/actuator/health
    curl -H "Accept: application/json" http://localhost:8081/actuator/health

then I get a 200 response.

If I downgrade SC OpenFeign to 2.0.0.M2, then the accept header is not required anymore. 

I have noticed also that all the endpoints which return a `@ResponseBody `String present the same behavior.

**Comment by ryanjbaxter** [comment]
Couple of things to try/change.

1.  Use Boot 2.0.1.RELEASE
2. There is no need to include the openfeign dependency in your dependency management section

**Comment by jjroldan1** [comment]
Hi, 

I only added it to dependency management because it is recommended here [1]... However, if I don't add the dependency management, the problem goes away for now.  

Finchley.RC1 manages webmvc to 5.0.4 while the one here goes up to 5.0.5. If I upgrade my parent to Boot 2.0.1.RELEASE, then it all works fine. Seems there is incompatibility between them.

Closing this, Thanks!

[1] https://cloud.spring.io/spring-cloud-openfeign/
